spec/notifier_spec.rb

```diff
@@ -544,5 +544,22 @@ def to_json(*)
       @airbrake.create_deploy({})
       expect(a_request(:post, deploy_endpoint)).to have_been_made.once
     end
+
+    describe "when a host contains subdirectories" do
+      let(:deploy_host) do
+        "https://airbrake.io/subdir/"
```

Comment:
Airbrake doesn't use paths for hosts. I assume the fix is needed for Errbit.

1. Would you mind changing `airbrake.io` to an Errbit URL, so it's more clear
2. `subdir` is a wrong term here, I would call this `subpath` since it has nothing to do with directories